generators/server/templates/src/main/docker/keycloak.yml.ejs

```diff
@@ -21,16 +21,16 @@ version: '<%= DOCKER_COMPOSE_FORMAT_VERSION %>'
 services:
   keycloak:
     image: <%= DOCKER_KEYCLOAK %>
-    command: ["-b", "0.0.0.0", "-Dkeycloak.migration.action=import", "-Dkeycloak.migration.provider=dir", "-Dkeycloak.migration.dir=/opt/jboss/keycloak/realm-config", "-Dkeycloak.migration.strategy=OVERWRITE_EXISTING", "-Djboss.socket.binding.port-offset=1000", "-Dkeycloak.profile.feature.upload_scripts=enabled"]
+    entrypoint: /bin/bash -c
+    command: ["/opt/keycloak/bin/kc.sh import --dir=/opt/jboss/keycloak/realm-config --override=true && /opt/keycloak/bin/kc.sh start-dev"]
     volumes:
       - ./realm-config:/opt/jboss/keycloak/realm-config
     environment:
-      - KEYCLOAK_USER=admin
-      - KEYCLOAK_PASSWORD=admin
-      - DB_VENDOR=h2
+      - KC_DB=dev-file
+      - KEYCLOAK_ADMIN=admin
+      - KEYCLOAK_ADMIN_PASSWORD=admin
+      - KC_FEATURES=scripts
     # If you want to expose these ports outside your dev PC,
     # remove the "127.0.0.1:" prefix
     ports:
-      - 127.0.0.1:9080:9080
-      - 127.0.0.1:9443:9443
-      - 127.0.0.1:10990:10990
+      - 127.0.0.1:9080:8080
```

Comment:
Is https (with 9443) no longer an option?

Reply:
It is, its just that the proxy type should be configured with env variables two...
https://www.keycloak.org/server/all-config#_proxy
I added it back but it won't work unless you change the config.
I also added some doc linking to this file, but I don't think we should explain this for each config option,